rpc_jobs/pull_request_whisperer.yml

```diff
@@ -0,0 +1,78 @@
+- project:
+    name: 'Pull-Request-Whisperer-Jobs'
+    series:
+      - all_branches:
+          branches: ".*"
+    repo:
+      - rpc_openstack:
+          repo_url: "https://github.com/rcbops/rpc-openstack"
+          run_issue_linker: true
+      - rpc_gating:
+          repo_url: "https://github.com/rcbops/rpc-gating"
+          run_issue_linker: true
+      - releases:
+          repo_url: "https://github.com/rcbops/releases"
+          run_issue_linker: true
+          comment_body: |
+            Thank you for pull request to `rcbops/releases`!
+
+            Once all `check` jobs have passed and the pull request has received a
+            sufficient amount of reviews, the repository's `release` jobs can be triggered by
```

Comment:
"Amount" sounds a little strange to me, maybe "number"?

Reply:
Fair enough, fixing.